js/views/attachment_view.js

```diff
@@ -25,7 +30,11 @@
         this.trigger('update');
       },
       render: function() {
-        this.$el.attr('src', this.dataUrl);
+        if (this.source == null) {
```

Comment:
Not sure if you saw the CI error, but JSHint doesn't like the double-equals here. Anyway, I think it could be simplified to `if (this.source) {`

Reply:
Yep! Fixed it in https://github.com/signalapp/Signal-Desktop/pull/2040/commits/161f759b9ccfb74b94c8cadcda5e0ca19f216edd. ESLint has a good `smart` option for `eqeqeq` rule as `typeof x === 'undefined' || x === null` can be safely shortened to `x == null`. I’ve just been bitten by automatic type coercions in the past so I err on the side of making them explicit.